internal/pkg/cli/app_init.go

```diff
@@ -82,6 +85,11 @@ func (opts *InitAppOpts) Validate() error {
 
 // Execute writes the application's manifest file and stores the application in SSM.
 func (opts *InitAppOpts) Execute() error {
+	projectName := viper.GetString(projectFlag)
```

Comment:
Is it possible that `projectName` is empty/invalid here?

Reply:
In the Validate function we make sure it's not empty or invalid